.github/workflows/build.yml

```diff
@@ -0,0 +1,18 @@
+name: compile-uradvd
+run-name: Compile uradvd
+
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+  schedule:
+    - cron:  '0 23 * * *'
```

Comment:
I don't think we use scheduled actions anywhere else... shouldn't it be sufficient to have this run for PRs and main?

Reply:
As you pointed out, the compilation is expected to fail in the future, once tooling changes. I'd like to have it fail in the CI as early as possible.